RELEASE.md

```diff
@@ -127,7 +126,7 @@ git push origin "${tag}" "pkg/apis/monitoring/${tag}" "pkg/client/${tag}"
 
 Signed tag with a GPG key is appreciated, but in case you can't add a GPG key to your Github account using the following [procedure](https://docs.github.com/articles/generating-a-gpg-key), you can replace the `-s` flag by `-a` flag of the `git tag` command to only annotate the tag without signing.
 
-Our CI pipeline will automatically push the container images to [quay.io](https://quay.io/organization/prometheus-operator) and [ghcr.io](https://github.com/prometheus-operator/prometheus-operator/pkgs/container/prometheus-operator)
+Once a tag is created, the release process through Github action will be triggered for this tag. Our CI pipeline will automatically push the container images to [quay.io](https://quay.io/organization/prometheus-operator) and [ghcr.io](https://github.com/prometheus-operator/prometheus-operator/pkgs/container/prometheus-operator). Wait till [publish](https://github.com/prometheus-operator/prometheus-operator/actions/workflows/publish.yaml) and [release](https://github.com/prometheus-operator/prometheus-operator/actions/workflows/release.yaml) jobs are completed so that assets will be attached to release correctly. We have observed in past that if we create a draft release and publish it later assets are not attached correctly hence its advised to wait till release job is completed to create the release.
```

Comment:
I feel that the last sentence should be moved to the next paragraph where we instruct how to create/publish the release.
```suggestion
Once a tag is created, the `publish` Github action will push the container images to [quay.io](https://quay.io/organization/prometheus-operator) and [ghcr.io](https://github.com/prometheus-operator/prometheus-operator/pkgs/container/prometheus-operator). Wait until the [publish](https://github.com/prometheus-operator/prometheus-operator/actions/workflows/publish.yaml) workflow is complete before going to the next step.
```